blog.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: plantillablog
+title: Blog
+permalink: /blog
+---
+
+<p>Estás visitando el blog de Antonio Calabuig!</p>
+
+<ul>
+    <li><a href="/blog/2024-03-05-a">Post A</a></li>
```

Comment:
Esto no es correcto: debes poner un listado a todos los posts, incluyendo los que puedas crear. Ejemplo:
https://jekyllrb.com/docs/posts/#displaying-an-index-of-posts